### ISSUE TITLE
TensorFlow logits.data() returning inconsistent probabilities

### ISSUE THREAD

**Issue by oninross** [tensorflow/tfjs#382]
I am following the steps from this project: [https://github.com/google/emoji-scavenger-hunt#build-your-own-model](url) and I managed to train models for the front end.  I can snap an image and return the probabilities.

The issue is actually the predictions/probabilities are inconsistent.  For example I only have Keyboard and headphones to classify and identify.

**1st time training:** Keyboard is identified as Headphones and Headphones is identified as Keyboard.
**2nd time training:** Keyboard is identified as Keyboard and Headphones is identified as Headphones.
**3rd time training:** Keyboard is identified as Headphones and Headphones is identified as Keyboard.

**Comment by oninross** [comment]
Problem solved.  I completely ignored the output labels text file completely.  I wasn't aware that the labels get scrambled after training and conversion.
